### ISSUE TITLE
i dont can use avocode u.u

### ISSUE THREAD

**Issue by SantiagoMolinaOrozco** [avocode/avocode#1070]
IMPORTANT: Please try to describe how did it happen.

-----
App v. `2.10.0`
Source UID: 87259
```
TypeError: Cannot read property 'dist' of null
    at res (/home/san/Programs/avocode/Avocode-linux-x64/resources/app.asar/node_modules/linux-distro/index.js:23:12)
    at /home/san/Programs/avocode/Avocode-linux-x64/resources/app.asar/node_modules/linux-distro/node_modules/getos/index.js:65:18
    at customLogic (/home/san/Programs/avocode/Avocode-linux-x64/resources/app.asar/node_modules/linux-distro/node_modules/getos/index.js:112:13)
    at /home/san/Programs/avocode/Avocode-linux-x64/resources/app.asar/node_modules/linux-distro/node_modules/getos/index.js:62:16
    at tryToString (fs.js:455:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:442:12)
```

**Comment by jakubzitny** [comment]
Hi @SantiagoMolinaOrozco, I'm so sorry you have this problem. This is a stupid bug we have to fix asap. It's related to your Linux distro. Could you please give us more information on the distribution and version of Linux you are using? Maybe even which desktop environment you have? It would help us with fixing this bug a lot.

For now, please temporarily downgrade to older `Avocode 2.9.1`, you can download the x-64 Linux version [here](https://media.avocode.com/download/avocode-app/2.9.1/avocode-app-linux-2.9.1.zip). If you need 32-bit or anything else, please let me know 😉

**Comment by SantiagoMolinaOrozco** [comment]
Operative sistem : arch,
Desktop enviroment : kde plasma 5,
Linux kernel version : 4.7.2-1-ARCH
In oficial Aur we have 2.9.1, i am use this temporarily

**Comment by jakubzitny** [comment]
Allright thanks a lot man, this will help.

By the way, do you happen to know if there is any way of checking that user has Arch Linux installed? Do you have a `/etc/arch-release` file in your OS or something similar that'd give similar output than usual `/etc/issue` or `lsb_release` output?

**Comment by SantiagoMolinaOrozco** [comment]
yes i have `/etc/arch-release` but this file is empty
i dont know how ckecking that user has arch linux installed

**Comment by jakubzitny** [comment]
Thanks @SantiagoMolinaOrozco, awesome :) This will be fixed for the next Avocode release. We'll keep you updated 😉

**Comment by SantiagoMolinaOrozco** [comment]
Your welcome, ;) i liked your job.

**Comment by jakubzitny** [comment]
Hi @SantiagoMolinaOrozco, this has been fixed in our recent release `2.10.1`⏩.

Please let us know if you have any more problems with Avocode.
